materr.cfg

```diff
@@ -0,0 +1,6 @@
+steps.Git(
+    repourl = ...,
+    method = 'clobber',
+    mode = 'full',
+    copycmd = ['rsync', '-av', '--delete'],
```

Comment:
Where do you see the option `copycmd` documented?

Reply:
sorry, i added it to run it  in solaris

Reply:
I mean: I didn't find anything called "`copycmd`" in the source code or documentation of Buildbot. When Python sees this option, how will it know what to do with it?

Reply:
The "copycmd" option is likely a custom option that is specific to the Python script being executed. When the script is executed, Python will recognize the "copycmd" option as a custom option and will pass it as an argument to the corresponding function or method that uses it to perform a file copy operation. It is up to the script developer to define and implement the behavior of the "copycmd" option within the script.